src/faq/faq.component.html

```diff
@@ -0,0 +1,68 @@
+<mat-accordion>
+  <mat-toolbar color="primary">Contexte :</mat-toolbar>
+  <mat-expansion-panel>
+    <mat-expansion-panel-header>Dans quel contexte cette application a-t-elle été développée ?</mat-expansion-panel-header>
+    <div>
```

Comment:
Remplacer les div par une autre palise 
Par exemple <p> 
Web Semantic is important